### DISCUSSION TITLE
How to see assembly when debugging firmware, (I dont have source code)

### DISCUSSION THREAD

**Discussion by aguaviva** [rohanrhu/gdb-frontend#63] [Debugging Q&A]
I can see the disassembly in the console window using the command `disass 0, 200` for example,  but the disassembly window is empty, how can  I make is show the disasm?

![image](https://github.com/rohanrhu/gdb-frontend/assets/5200915/b10563de-d81f-4c14-809d-51bb6704590e)

Thanks for this awesome software!

**Comment by rohanrhu** [comment]
Ummm as I remember, the disassembly view currently shows instructions of current frame. When the bounds of current space is not known (for instance, whe you don't have debug symbols), it shows nothing.

When you do `disass 0, 200`, you are specifying the bound offsets of instructions to show. I think we can add this feature to the disassembly view so we'll be able to specify custom bound offsets and see instructions in disassembly view.

**Reply by aguaviva** [reply]
I'd be great if, when there is no frame, it could disassemble from the current ip onwards. That already would be a big help :)
